contracts/Gate.sol

```diff
@@ -165,8 +216,22 @@ contract Gate is ProtocolVersioned, Owned {
         return stakeRequestAmount;
     }
 
-    /// @dev In order to accept stake the staker needs to approve gate contract for bounty amount.
-    ///      This can be called only by whitelisted worker address
+    /**
+      * @notice external function to accept requested stake.
+      *
+      * @dev In order to accept stake the staker needs to approve gate contract for bounty amount.
```

Comment:
I think this comment needs some improvement:
- it is not clear to me what is doing the accepting in the clause: "In order to accept stake"
- I think it's the worker/the caller of this function that needs to approve the Gate contract for the bounty, not the staker
- I would switch the last two lines, "OpenSTProtocol is approved..." and "Bounty amount is transferred...", so that they appear in the same order in which they occur in the function
- while I'm not sure if it's better to refer to "OpenSTProtocol" or "OpenSTValue", the variable name from `ProtocolVersioned` is lower-case, "openSTProtocol", so suggest sticking with that in the documentation comment

Reply:
There's a comment on what is currently L257 that has a "form" that should be "from": "// Transfer bounty amount form worker to gate contract"